plugins/in_exec/in_exec.c

```diff
@@ -119,15 +207,40 @@ static int in_exec_config_read(struct flb_in_exec_config *exec_config,
 {
     char *cmd = NULL;
     char *pval = NULL;
+    char *follow = NULL;
+    int argc = 0;
 
     /* filepath setting */
     cmd = flb_input_get_property("command", in);
     if (cmd == NULL) {
         flb_error("[in_exec] no input 'command' was given");
         return -1;
     }
-    exec_config->cmd = cmd;
 
+    /* split into an argv array */
+    char *arg = NULL;
+    for (arg = strtok_r(cmd, " ", &cmd); arg != NULL; arg = strtok_r(NULL, " ", &cmd)) {
+        exec_config->argv = flb_realloc(exec_config->argv, (argc + 2) * sizeof(char *));
+        exec_config->argv[argc++] = arg;
+    }
+    exec_config->argv[argc] = NULL;
+
+    /* follow setting */
+    follow = flb_input_get_property("follow", in);
+    if (follow != NULL) {
+        if (strcasecmp(follow, "true") == 0) {
+            exec_config->follow = 1;
```

Comment:
We defined FLB_TRUE/FLB_FALSE at flb_macros.h. Would you replace it?

Reply:
I updated the code to make use of the FLB_TRUE/FLB_FALSE macros.